packages/ifos/src/constants/ifos/arb.ts

```diff
@@ -1,3 +1,125 @@
+  {
+    id: 'testCrossChainIfo4',
+    version: 8,
+
+    address: '0x41cE6614F310315A43942fa6eCEdc19E93E50E63', // Test IFO
+    plannedStartTime: 1726632000,
+
+    isActive: true,
+    name: 'Test Arbitrum IFO',
+    description: 'This is an IFO on Arbitrum to test cross-chain IFOs functionality on PancakeSwap.',
+    currency: arbitrumTokens.cake,
+    token: arbitrumTokens.usdc,
+    articleUrl: 'https://pancakeswap.medium.com/',
+    campaignId: 'random-for-testing',
+    poolBasic: {
+      raiseAmount: '$10,000',
+    },
+    poolUnlimited: {
+      raiseAmount: '$90,000',
+      additionalClaimingFee: false,
+    },
+    tokenOfferingPrice: 0.1,
+    vestingTitle: 'testCrossChainIfo4',
+  },
+  {
+    id: 'testCrossChainIfo3',
+    version: 8,
+
+    address: '0xb7AdE79F9c517C9b3cc45E3522eBc4dCe1C54e7D', // Test IFO
+    plannedStartTime: 1726552801,
+
+    isActive: false,
+    name: 'Test Arbitrum IFO',
+    description: 'This is an IFO on Arbitrum to test cross-chain IFOs functionality on PancakeSwap.',
+    currency: arbitrumTokens.cake,
+    token: arbitrumTokens.usdc,
+    articleUrl: 'https://pancakeswap.medium.com/',
+    campaignId: 'random-for-testing',
+    poolBasic: {
+      raiseAmount: '$10,000',
+    },
+    poolUnlimited: {
+      raiseAmount: '$90,000',
+      additionalClaimingFee: false,
+    },
+    tokenOfferingPrice: 0.1,
+    vestingTitle: 'testCrossChainIfo3',
+  },
+  {
+    id: 'testCrossChainIfo2',
+    version: 8,
+
+    address: '0xD134a41Ac8c54bD8713F8B9307ded4FFF340E5C1', // Test IFO
+    plannedStartTime: 1726466400,
+
+    isActive: false,
+    name: 'Test Arbitrum IFO',
+    description: 'This is an IFO on Arbitrum to test cross-chain IFOs functionality on PancakeSwap.',
+    currency: arbitrumTokens.cake,
+    token: arbitrumTokens.usdc,
+    articleUrl: 'https://pancakeswap.medium.com/',
+    campaignId: 'random-for-testing',
+    poolBasic: {
+      raiseAmount: '$10,000',
+    },
+    poolUnlimited: {
+      raiseAmount: '$90,000',
+      additionalClaimingFee: false,
+    },
+    tokenOfferingPrice: 0.1,
+    vestingTitle: 'testCrossChainIfo2',
+  },
+  {
+    id: 'testCrossChainIfo1',
+    version: 8,
+
+    address: '0x7022f1335664D54C7D1bC22CFDa02f2C16CeEB7f', // Test IFO
+    plannedStartTime: 1726236000,
+
+    isActive: false,
+    name: 'Test Arbitrum IFO',
+    description: 'This is an IFO on Arbitrum to test cross-chain IFOs functionality on PancakeSwap.',
+    currency: arbitrumTokens.cake,
+    token: arbitrumTokens.usdc,
+    articleUrl: 'https://pancakeswap.medium.com/',
+    campaignId: 'random-for-testing',
+    poolBasic: {
+      raiseAmount: '$10,000',
+    },
+    poolUnlimited: {
+      raiseAmount: '$90,000',
+      additionalClaimingFee: false,
+    },
+    tokenOfferingPrice: 0.1,
+    vestingTitle: 'testCrossChainIfo1',
+  },
+  {
+    id: 'testCrossChainIfo0',
+    version: 8,
+
+    address: '0x6164B999597a6F30DA9aEF8A7F31D6dD7AE57e04', // Test IFO
```

Comment:
is this test config going to be used often and wil the addresses of the test ifo contracts change much? if so would it be better to make a maping for these test contract addresses for each chain and use the address here, that way its easier to keep track of the contracts and change them if nessecary

Reply:
that's a good idea. we can consider next time, as right now not many more test IFOs

Reply:
yeah i was kinda thinking that. just wanted to comment just incase. cool

Reply:
yeah sure. i can take a note a raise pr in future